app/components/containerInspect/containerInspect.html

```diff
@@ -2,17 +2,25 @@
   <rd-header-title title="Container inspect">
   </rd-header-title>
   <rd-header-content>
-    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: containerInfo.Id})">{{ containerInfo.Name|trimcontainername }}</a> &gt; Inspect
+    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: containerInfo.object.Id})">{{ containerInfo.object.Name|trimcontainername }}</a> &gt; Inspect
   </rd-header-content>
 </rd-header>
 
 <div class="row">
   <div class="col-lg-12 col-md-12 col-xs-12">
     <rd-widget>
       <rd-widget-header icon="fa-icon-circle" title="Inspect">
+        <div class="btn-group" role="group" aria-label="...">
+          <button class="btn btn-sm btn-success" ng-click="toggle()">
+            <span ng-show="TextView"><i class="fa fa-hand-pointer-o space-right" aria-hidden="true"></i>Tree</span>
+            <span ng-hide="TextView"><i class="fa fa-file-text-o space-right" aria-hidden="true"></i>Text</span>
```

Comment:
In Portainer, button groups are usually used to switch between options unless there is not much of them so I think that if we detach mode to separate button group it should be simpler in terms of UX and will be similar to other places.

<img width="1015" alt="screen shot 2017-10-23 at 11 00 37" src="https://user-images.githubusercontent.com/6943514/31878083-7db01d28-b7e1-11e7-948e-898a24eb28f5.png">